frontend/tests/utils/downloadUtils.ts

```diff
@@ -11,6 +11,7 @@ import sharp from "sharp";
 import { goToWMG } from "./wmgUtils";
 import { CELL_TYPE_ROW_CLASS_NAME } from "src/views/WheresMyGeneV2/components/HeatMap/components/YAxisChart/constants";
 import { tryUntil } from "tests/utils/helpers";
+import { getCurrentDate } from "src/views/WheresMyGeneV2/components/GeneSearchBar/components/SaveExport/utils";
```

Comment:
extract `getCurrentDate` to a `utils.ts` file, so it can be imported in test files